### ISSUE TITLE
Multiple issues packaging for Fedora

### ISSUE THREAD

**Issue by eeickmeyer** [Houston4444/RaySession#40]
Please see the [this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1801352#c12) regarding this application, particularly points 2-5. Seems as though you might be doing some stuff that isn't allowed, and you might want to think about making some manpages.

Additionally, I've been getting a lot of flack for having packaged this for Ubuntu and that this project simply shouldn't exist, probably due to fragmentation fears.

**Comment by Houston4444** [comment]
thanks for this feedback
.
Regarding the 5 points:
1 ) REQUIRES
 RS uses git and wmctrl (not xdotool, but maybe wmctrl is in xdotool on Fedora). These 2 programs are not needed, RS works without them. However, if the use of wmctrl is quite trivial, the use of git is recommended. Jack connection kit is only used by the ray-jackpatch (connections memory) which is a subprogram, it should not be considered as a dependency.

2) 
"snapshot explain", ok, I'll replace the blank.
"Ray Sessions" folder, I need more reflection, this name was based on the template of NSM ("NSM Sessions" folder). For RaySession, this directory string is translatable. Shouldn't it and why ?

3)
Globally, Manual is the most missing feature in RaySession.  Maybe man pages could only take the stdout of "command --help" in a first time.

4 )
Possible. I did try RS under Wayland (plasma wayland), but a long time ago and not a long time.
If someone is motivated to write bug report about this, It would be appreciated.

5)
No Idea. Before to change anything here, I need to be 100% sure this is strictly forbidden.


> Additionally, I've been getting a lot of flack for having packaged this for Ubuntu and that this project simply shouldn't exist, probably due to fragmentation fears.

I have got a little idea of who tells that. However, due to its regressive behavior, this discussion doesn't interest me.

**Comment by eeickmeyer** [comment]
Thanks for the response.

1) I made all three of those Recommends. Means they're installed if available but if not available, no big deal.

2) I am renaming "snapshots explain" to "snapshots-explain" as part of the prep process with this issue linked/commented.

3) Likewise, I'm linking this comment in that the manpage issue has been reported.

4) I ran it under Xorg and found no issues, so I'm guessing it's a Wayland thing and we need not bother too much with this.

5) I verified that it's not strictly forbidden, so don't make any changes at this time.
